tests/slow/src/test/scala/tests/feature/Scala3CodeActionLspSuite.scala

```diff
@@ -260,6 +260,163 @@ class Scala3CodeActionLspSuite
        |""".stripMargin,
   )
 
+  check(
+    "extract-if-cond",
+    """|object Main:
+       |  def main(i : Int) =
+       |    if(2 > <<3>>) 5 else 4
+       |
+       |""".stripMargin,
+    s"""|${ExtractValueCodeAction.title("2 > 3")}
+        |""".stripMargin,
+    """|object Main:
+       |  def main(i : Int) =
+       |    val newValue = 2 > 3
+       |    if(newValue) 5 else 4
+       |
+       |""".stripMargin,
+  )
```

Comment:
General question about those tests: shouldn't we add just 1 test for extract value here to see if it works with Scala3 and move the rest to the `unit` ones?
cc: @tgodzik

Reply:
[Project structure](https://scalameta.org/metals/docs/contributors/getting-started#project-structure) section in docs isn't helpful about slow tests